packages/knip/src/plugins/vite/index.ts

```diff
@@ -12,6 +18,52 @@ const isEnabled: IsPluginEnabled = ({ dependencies }) => hasDependency(dependenc
 
 export const config = ['vite.config.{js,mjs,ts,cjs,mts,cts}'];
 
+const hasReactBabelPlugins = (config: ViteConfig) => {
```

Comment:
If we can move it over to the vitest plugin, then this helper could go the `helpers.ts`